src/NosCore.Algorithm/Constants.cs

```diff
@@ -9,6 +9,7 @@ internal class Constants
         internal const byte MaxFairyLevel = 80;
         internal const byte MaxJobLevel = 80;
         internal const byte MaxHeroLevel = 60;
-        internal static readonly int ClassCount = Enum.GetNames(typeof(CharacterClassType)).Length;
```

Comment:
this has nothing to do in that file it's not something global but something particular for a specific implementation